src/tools/tests/manifest2proto-test.ts

```diff
@@ -712,6 +714,56 @@ describe('manifest2proto', () => {
     });
   });
 
+  it('encodes max variable type - writeSuperset constraint', async () => {
+    const constraint = EntityType.make(['Foo'], {value: 'Text'}).singletonOf();
+    const varType = TypeVariable.make('a', constraint, null, true);
+    assert.deepStrictEqual(await toProtoAndBackType(varType), {
+      variable: {
+        name: 'a',
+        maxType: true,
+        constraint: {constraintType: {
+            singleton: {singletonType: {
```

Comment:
This indentation is weird. Either do it one element per line, e.g.

```
a: {
  b: {
    c: 'foo'
  }
}
```

or inline sometimes while ensuring that the next indent is +2 spaces:

```
a: {b: {
  c: 'foo'
}}
```

But don't mix it in a way that makes following indent of arbitrary depth.

Reply:
This style was adapted after a suggestion from Myk, see: https://github.com/PolymerLabs/arcs/pull/5302#discussion_r425501978

Is there a middle ground that we can settle on?

Reply:
This indentation is also consistent w/ the rest of the tests in this file (and others).

Reply:
In Myk's suggestion every following level of indentation is 2 spaces, I am happy with more compactness, but I don't like this:
```
a: {b: {c: {d: {
        x: 'foo'
      }
    }
  }
}
```

Reply:
Ah, I see what you mean about mixing. Cool, it's clear that this should change and how it should change.

Reply:
Huh, the view in my IDE is different.

Reply:
No, the file is just really inconsistent -- I'll take the liberty of cleaning it up, generally.